tests/func/test_cli.py

```diff
@@ -175,11 +175,17 @@ class A:
             Cmd(args)
 
 
-def test_unknown_command_help(capsys):
+def test_unknown_command_help(caplog, capsys):
     try:
         _ = parse_args(["unknown"])
     except DvcParserError:
         pass
+    assert (
+        "dvc: 'unknown' is not a dvc command. See 'dvc --help'" in caplog.text
+    )
```

Comment:
Test expectations changed here since we're not printing the full usage statement for unknown commands.